docker/entrypoint

```diff
@@ -61,7 +61,8 @@ if [ "${AUTO_UPDATE:-}" != "true" ]; then
   AUTO_UPDATE="false"
 fi
 
-SNO_RUN_PARAMS="${RUN_PARAMS} --console.address=:14002"
+: ${STORJ_CONSOLE_ADDRESS:=0.0.0.0:14002}
+SNO_RUN_PARAMS="${RUN_PARAMS}"
```

Comment:
shouldn't we have here something like `${RUN_PARAMS} --${STORJ_CONSOLE_ADDRESS}`

Reply:
or `${RUN_PARAMS} --console.address=${STORJ_CONSOLE_ADDRESS}`

Reply:
ah I forgot that STORJ_CONSOLE_ADDRESS will be used in runtime